### PR TITLE
proxy-from-env: Add missing URL type for input parameter

### PR DESCRIPTION
**Pull request by flopes89**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Rob--W/proxy-from-env/blob/master/index.js#L24

### CODE REVIEW

**Review by JasonHK — APPROVED** [review]

**Review by JasonHK — CHANGES_REQUESTED** [review]
Your build was failed, please check again

File: types/proxy-from-env/index.d.ts [hunk 1]
```diff
@@ -10,4 +10,4 @@
  * @returns The URL of the proxy that should handle the request to the given
  *          URL.
  */
-export function getProxyForUrl(url: string): string;
+export function getProxyForUrl(url: string|URL): string;
```
Comment (JasonHK) [inline types/proxy-from-env/index.d.ts]: The URL object `getProxyForUrl` accept is [this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d129b9b777901659b1559f9355f9548496e72a2b/types/node/url.d.ts#L20), not the standard `URL`.

**Review by JasonHK — CHANGES_REQUESTED** [review]

File: types/proxy-from-env/index.d.ts (hunk 1 above)
Comment (flopes89) [reply types/proxy-from-env/index.d.ts]: I updated the types to use the `url` type, but the tests fail telling me the `"url"` module cannot be found. How can I fix this so the import works as expected? I checked other type definitions and they do this exactly the same as I do? (e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/parseurl/index.d.ts#L9 )

**Comment by flopes89** [comment]
I updated the types to use the url type, but the tests fail telling me the "url" module cannot be found. How can I fix this so the import works as expected? I checked other type definitions and they do this exactly the same as I do? (e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/parseurl/index.d.ts#L9 )

File: types/proxy-from-env/index.d.ts [hunk 2]
```diff
@@ -1,13 +1,16 @@
 // Type definitions for proxy-from-env 1.0
 // Project: https://github.com/Rob--W/proxy-from-env#readme
 // Definitions by: JasonHK <https://github.com/JasonHK>
+//                 FloPes <https://github.com/flopes89>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import { Url } from "url";
```
Comment (JasonHK) [inline types/proxy-from-env/index.d.ts]: @flopes89 You're missing the triple-slash directive. Please put the code below before line 7.

```typescript
/// <reference types="node" />
```

**Review by JasonHK — CHANGES_REQUESTED** [review]

**Comment by flopes89** [comment]
Thanks for the hint! I updated the reference type. Sorry for the mistake.

**Review by JasonHK — APPROVED** [review]
Thank you!

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**